test/NuGet.Clients.Tests/NuGet.CommandLine.Test/RestoreNETCoreTest.cs

```diff
@@ -6124,23 +6124,23 @@ public async Task RestoreNetCore_InteropTypePackage()
         [Fact]
         public async Task RestoreNetCore_MultiTFM_ProjectToProject_PackagesLockFile()
         {
-            // Arrange
+            // Arrange	
             using (var pathContext = new SimpleTestPathContext())
             {
-                // Set up solution, project, and packages
+                // Set up solution, project, and packages	
```

Comment:
trailing whitespace makes me sad 😢

Reply:
I formatted the whole file and that's what the formatter gave me :)
I will go through it and remove any unnecessary changes.

Reply:
fixed.